config/manifests/bases/argocd-operator.clusterserviceversion.yaml

```diff
@@ -660,5 +660,5 @@ spec:
   maturity: alpha
   provider:
     name: Argo CD Community
-  replaces: argocd-operator.v0.1.0
-  version: 0.2.0
+  replaces: argocd-operator.v0.2.0
+  version: 0.0.0
```

Comment:
should be 0.3.0 ?

Reply:
It doesn't matter, it gets replaced in the generated CSV by the correct version.